UIntPtrGetHashCode.csproj and IntPtrGetHashCode.csproj have build problems

The build error were introduced in https://github.com/dotnet/coreclr/pull/3511/files

intptrgethashcode.cs(117,17): error CS0103: The name 'retVal' does not exist in the current context [E:\git\coreclr\tests\src\CoreMangLib\cti\system\intptr\IntPtrGetHashCode.csproj]
intptrgethashcode.cs(125,24): error CS0103: The name 'retVal' does not exist in the current context [E:\git\coreclr\tests\src\CoreMangLib\cti\system\intptr\IntPtrGetHashCode.csproj]
intptrgethashcode.cs(129,52): error CS0103: The name 'id' does not exist in the current context [E:\git\coreclr\tests\src\CoreMangLib\cti\system\intptr\IntPtrGetHashCode.csproj]
intptrgethashcode.cs(129,140): error CS0103: The name 'i' does not exist in the current context [E:\git\coreclr\tests\src\CoreMangLib\cti\system\intptr\IntPtrGetHashCode.csproj]
intptrgethashcode.cs(130,17): error CS0103: The name 'retVal' does not exist in the current context [E:\git\coreclr\tests\src\CoreMangLib\cti\system\intptr\IntPtrGetHashCode.csproj]
intptrgethashcode.cs(136,13): error CS0103: The name 'retVal' does not exist in the current context [E:\git\coreclr\tests\src\CoreMangLib\cti\system\intptr\IntPtrGetHashCode.csproj]
intptrgethashcode.cs(138,16): error CS0103: The name 'retVal' does not exist in the current context [E:\git\coreclr\tests\src\CoreMangLib\cti\system\intptr\IntPtrGetHashCode.csproj]
uintptrgethashcode.cs(169,48): error CS1503: Argument 1: cannot convert from 'long' to 'uint' [E:\git\coreclr\tests\src\CoreMangLib\cti\system\uintptr\UIntPtrGetHashCode.csproj]
uintptrgethashcode.cs(170,48): error CS1503: Argument 1: cannot convert from 'long' to 'uint' [E:\git\coreclr\tests\src\CoreMangLib\cti\system\uintptr\UIntPtrGetHashCode.csproj]
uintptrgethashcode.cs(174,17): error CS0103: The name 'retVal' does not exist in the current context [E:\git\coreclr\tests\src\CoreMangLib\cti\system\uintptr\UIntPtrGetHashCode.csproj]
uintptrgethashcode.cs(182,24): error CS0103: The name 'retVal' does not exist in the current context [E:\git\coreclr\tests\src\CoreMangLib\cti\system\uintptr\UIntPtrGetHashCode.csproj]
uintptrgethashcode.cs(186,52): error CS0103: The name 'id' does not exist in the current context [E:\git\coreclr\tests\src\CoreMangLib\cti\system\uintptr\UIntPtrGetHashCode.csproj]
uintptrgethashcode.cs(186,140): error CS0103: The name 'i' does not exist in the current context [E:\git\coreclr\tests\src\CoreMangLib\cti\system\uintptr\UIntPtrGetHashCode.csproj]
uintptrgethashcode.cs(187,17): error CS0103: The name 'retVal' does not exist in the current context [E:\git\coreclr\tests\src\CoreMangLib\cti\system\uintptr\UIntPtrGetHashCode.csproj]
uintptrgethashcode.cs(193,13): error CS0103: The name 'retVal' does not exist in the current context [E:\git\coreclr\tests\src\CoreMangLib\cti\system\uintptr\UIntPtrGetHashCode.csproj]
uintptrgethashcode.cs(195,16): error CS0103: The name 'retVal' does not exist in the current context [E:\git\coreclr\tests\src\CoreMangLib\cti\system\uintptr\UIntPtrGetHashCode.csproj]

--- a/tests/src/CoreMangLib/cti/system/intptr/intptrgethashcode.cs
+++ b/tests/src/CoreMangLib/cti/system/intptr/intptrgethashcode.cs
@@ -104,7 +104,7 @@ public class IntPtrGetHashCode
     
     public bool PosTest4()
     {
-        bool retValue = true;
+        bool retVal = true;
         try
         {
             long addressOne = 0x123456FFFFFFFFL;
@@ -126,7 +126,7 @@ public class IntPtrGetHashCode
             }
             else
 		   	{
-                TestLibrary.TestFramework.LogError(id, String.Format("IntPtr should not have thrown an OverflowException for value {0}: ", i) + ex.ToString());
+                TestLibrary.TestFramework.LogError("004", "IntPtr should not have thrown an OverflowException: " + ex.ToString());
                 retVal = false;
 		   	}
         }

--- a/tests/src/CoreMangLib/cti/system/uintptr/uintptrgethashcode.cs
+++ b/tests/src/CoreMangLib/cti/system/uintptr/uintptrgethashcode.cs
@@ -161,11 +161,11 @@ public class UIntPtrGetHashCode
     
     public bool PosTest4()
     {
-        bool retValue = true;
+        bool retVal = true;
         try
         {
-            long addressOne = 0x123456FFFFFFFFL;
-            long addressTwo = 0x654321FFFFFFFFL;
+            ulong addressOne = 0x123456FFFFFFFFUL;
+            ulong addressTwo = 0x654321FFFFFFFFUL;
             System.UIntPtr ipOne = new UIntPtr(addressOne);
             System.UIntPtr ipTwo = new UIntPtr(addressTwo);
             if (ipOne.GetHashCode() == ipTwo.GetHashCode())
@@ -183,7 +183,7 @@ public class UIntPtrGetHashCode
             }
             else
 		   	{
-                TestLibrary.TestFramework.LogError(id, String.Format("IntPtr should not have thrown an OverflowException for value {0}: ", i) + ex.ToString());
+                TestLibrary.TestFramework.LogError("004", "IntPtr should not have thrown an OverflowException: " + ex.ToString());
                 retVal = false;
 		   	}
         }

